### PR TITLE
Add class invoice

### PR DESCRIPTION
**Pull request by aytouna**

### CODE REVIEW

File: src/main/java/net/dcess/gestrec/entity/Invoice.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotEmpty;
+
+/**
+ *
+ * @author Nexxion Sarl
+ */
```
Comment (JalalSordo) [inline src/main/java/net/dcess/gestrec/entity/Invoice.java:18]: missing:
@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity

**Review by JalalSordo — CHANGES_REQUESTED** [review]
rework required

File: src/main/java/net/dcess/gestrec/entity/Invoice.java (hunk 1 above)
Comment (aytouna) [reply src/main/java/net/dcess/gestrec/entity/Invoice.java:18]: @ouiameTh

**Review by JalalSordo — APPROVED** [review]
will fix it myself

**Pull request merged**